### ISSUE TITLE
Mismatch between pip version name and odml.__version__ in 1.4.4 release

### ISSUE THREAD

**Issue by morales-gregorio** [G-Node/python-odml#349]
Hey!

Just noticed that while the latest release returns version `1.4.4`, 
```
>>> import odml
>>> odml.__version__
'1.4.4'
```
pip does not recognize this and is expecting `1.4.4.2` instead:
```
Pip subprocess error:
ERROR: Could not find a version that satisfies the requirement odml==1.4.4 (from -r /users/morales/V1_V4_1024_electrode_resting_state_data/scripts/python_scripts/.snakemake/conda/condaenv.cnh95bm5.requirements.txt (line 8)) (from versions: 1.0, 1.1, 1.2, 1.2.1, 1.3.0, 1.3.1, 1.3.2, 1.3.3, 1.3.4.1, 1.4.0.3, 1.4.1.1, 1.4.2.1, 1.4.3.2, 1.4.4.2)
ERROR: No matching distribution found for odml==1.4.4 (from -r /users/morales/V1_V4_1024_electrode_resting_state_data/scripts/python_scripts/.snakemake/conda/condaenv.cnh95bm5.requirements.txt (line 8))
```

which has caused me some problems when trying to recreate a conda environment from an exported `.yml` file, that I have to manually adjust.

This is just some minor issue, but can lead to some unnecessary confusion.

All best,
Aitor

**Comment by mpsonntag** [comment]
hi @morales-gregorio! thanks for this issue, I was not aware of this problem. I think the only way to properly resolve this is doing another release where the actual name of the file uploaded to PyPI contains the exact version number.

**Comment by mpsonntag** [comment]
hi @morales-gregorio, I just released a new odml version (`1.4.5`) and also uploaded the released version to PyPI. The package version number should now exactly match the version number that is available via the PyPI index. I'll keep this issue in mind for any future releases.
Please let us know, if the issue has been resolved with the new release.

**Comment by morales-gregorio** [comment]
Thanks @mpsonntag the issue is indeed solved now!
